### PR TITLE
gem SecondBase

### PR DESCRIPTION
**Pull request by workgena**
- [x] install ``gem 'secondbase'`
- [x] fix broken task `db:structure:dump`
- [x] ActiveAdmin pages broke structure:dump. For example "ActiveAdmin::DatabaseHitDuringLoad: Your file, app/admin/routing/routing_groups.rb (line 9), caused a database error while Active Admin was loading."
- [x] `seeds.rb` to populate default application data (INSERT and pgq SELECT)
- [x] prepare test environment
- [x] rewrite Travis config
- [x] write README.md for SecondBase usage
- [x] Clean up `database.yml.*` (2 files)
- [x] Remove directories `aux` and `sql`. Fix debian-pakege and Makefile

### CODE REVIEW

File: app/models/cdr/base.rb [hunk 1]
```diff
@@ -1,7 +1,4 @@
 class Cdr::Base < Yeti::ActiveRecord
   self.abstract_class = true
-  establish_connection "#{Rails.env}_cdr".to_sym
-
-
```
Comment (Fivell) [inline app/models/cdr/base.rb]: @workgena DB_VER constant was used for performance

File: app/models/cdr/cdr.rb [hunk 2]
```diff
@@ -185,8 +183,10 @@ def status_sym
   end
 
 
+  # TODO: use DisconnectInitiator#display_name instead
+  # Why in was cached this way? `DISCONNECTORS = DisconnectInitiator.all.index_by(&:id)`
   def disconnect_initiator_name
-    DISCONNECTORS[self.disconnect_initiator_id].try(:name)
```
Comment (Fivell) [inline app/models/cdr/cdr.rb]: @workgena that was done for performance

File: app/models/cdr/cdr.rb [hunk 3]
```diff
@@ -185,8 +183,10 @@ def status_sym
...
+    disconnect_initiator.try(:name)
```
Comment (Fivell) [inline app/models/cdr/cdr.rb]: @workgena @dmitry-sinina  maybe we can use constant as we did before, I think we can make it lazy if we need, 

http://greyblake.com/blog/2014/10/05/lazy-object-pattern-in-ruby/
```
class LazyObject < ::BasicObject
  def initialize(&callable)
    @callable = callable
  end

  def __target_object__
    @__target_object__ ||= @callable.call
  end

  def method_missing(method_name, *args, &block)
    __target_object__.send(method_name, *args, &block)
  end
end


```

and then we can 

```
DISCONNECTORS = LazyObject.new { DisconnectInitiator.all.index_by(&:id) } 
```

Comment (workgena) [inline app/models/cdr/cdr.rb]: Currently it makes only one request(from localhost):
```sh
DisconnectInitiator Load (1.5ms)  SELECT "disconnect_initiators".* FROM "disconnect_initiators" WHERE "disconnect_initiators"."id" IN (3, 1, 0)
```

File: app/models/cdr/cdr.rb (hunk 2 above)
Comment (workgena) [inline app/models/cdr/cdr.rb]: Done, with new commit

File: app/models/cdr/base.rb (hunk 1 above)
Comment (workgena) [inline app/models/cdr/base.rb]: Did Lazy for DB_VER, in last commit

**Pull request merged**